src/helpers/formatCreationDate.js

```diff
@@ -0,0 +1,17 @@
+export default function formatCreationDate(totalMinutes) {
```

Comment:
1. Let's place all numbers to appropriate constants :)
2. Let's use AF approach for utils functions as well